plugins/generator-1.19.2/forge-1.19.2/procedures/set_world_border.java.ftl

```diff
@@ -0,0 +1,5 @@
+public static void setWorldBorder(int x, int y, int z, int size) {
```

Comment:
Why do you made a function? It won't execute the code when the procedure block is called. Did you even test your PR?